docs/source/installation.rst

```diff
@@ -81,11 +81,11 @@ After that, you can install the package in development mode
 
     pip install -e ".[dev]"
 
-To compile the Java classes (at the beginning or after changes), run
+To compile the Rust code (at the beginning or after changes), run
 
 .. code-block:: bash
 
-    python setup.py build_ext
+    python setup.py install
```

Comment:
I always use `install`, but can change it back to `build_ext` if that's preferred?

Reply:
Yeah, for compiling rust only changes `build_ext` might be a better option.

Reply:
This segment mainly applies to building rust changes only. But generally for dev we do recommend pip install -e . ".[dev]" which does install those dependencies.

Reply:
Thanks!